ui/src/apps/searchLogs/components/SearchHistory.tsx

```diff
@@ -134,20 +127,17 @@ export default function SearchHistory() {
   }
 
   const historyList: History[] = taskGroups.map((taskGroup) => {
-    const {
-      timeRange,
-      logLevel,
-      components,
-      searchValue,
-    } = parseSearchingParams(taskGroup)
-    const taskGroupID = taskGroup.task_group?.id || 0
+    const { timeRange, logLevel, stats, searchValue } = parseHistoryStatsParams(
```

Comment:
I think `parseHistoryStatsParams` can be simply removed. It doesn't do anything useful now.